Updated setup documentation on Nextcloud integration

marcalcobe opened the pull request:
The complete documentation on how to setup your Nextcloud documentation using the OAuth values step by step.
Also created the user-guide documentation md where I will be working next.

@psatyal could you please review the documentation from a language perspective?
@wielinde could you please review that the step by step is clear? Do we want to test this docs with someone totally disconnected from the Nextcloud integration design and development to see if it works as intended?

https://community.openproject.org/projects/openproject/work_packages/41752/activity

marcalcobe committed:
Updated setup documentation on Nextcloud integration

The complete documentation on how to setup your Nextcloud documentation using the OAuth values step by step.

Also created the user-guide documentation md where I will be working next.

psatyal commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,101 @@
 ---
 
-# OpenProject and Nextcloud integration 
+<<<<<<< Updated upstream
```
These lines with <<<< are a result of errors related to GitHub fetching/merging, I think.

psatyal committed:
Made grammatic and syntactical changes

Changed phrasing in a few places to make the clearer and sometimes more direct, restyled all references to buttons or actions in bold (to respect current style), change "IMPORTANT" text to quoted text (current style).

psatyal reviewed (commented):
Hi @marcalcobe. I've reviewed the system doc and changed phrasing in a few places to make the text clearer and sometimes more direct, restyled all references to buttons or actions in bold and changed "IMPORTANT" bits to quoted style (to respect current styling). Could you review to make sure that my re-structuring of certain sentences does not change the meaning/details of the integration?

I did not review the user guide bit yet because it seems like it's still work in progress.

marcalcobe commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -6,85 +6,81 @@ description: Nextcloud integration setup
 **2. Introduce the basic information of your integration** - in OpenProject
 
-As an administrator, in the new browser tab please open the File storages from the Administration page. To start with the configuration of your new Nextcloud integration click on the "+" button to add a new storage.
+Make sure you are logged in as an administrator in your OpenProject instance. Navigate to *File storages* from the Administration page. To start configuring new Nextcloud integration, click on the **Add** (+) button to add a new file storage.
```
Hi @psatyal,
I think a "your" is missing after the "configuring".

marcalcobe commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -6,85 +6,81 @@ description: Nextcloud integration setup
 **4. Generate and copy the Nextcloud OAuth values** - in Nextcloud and OpenProject
 
-As in the previous step the OAuth values are generated automatically, but this time the Nextcloud OAuth are created in the step 3 of the Nextcloud OpenProject Integration settings. This values are the ones that will allow the connection From Nextcloud to OpenProject, therefore, again, you need to copy them but this time from Nextcloud to OpenProject (without closing the Nextcloud one). If you hav ...
+As in the previous step, the OAuth values are generated automatically, but this time on the Nextcloud end. These are needed to allow the connection From Nextcloud to OpenProject to function, so you will need to once again copy them. Copy these values from Nextcloud to OpenProject (without closing the Nextcloud tab). If you had not clicked on the *Done. Continue setup* button from in previous step ...
```
Hi @psatyal,
In the last sentence after *Done. Continue setup* button, is it correct "from in previous step"? Shouldn't it be something like "in the previous step"?

marcalcobe commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -6,85 +6,81 @@ description: Nextcloud integration setup
+Now that the integration setup is complete and ready to use, there is just one more step for you the administrator: you need to activate the *File storages* module and specify for which projects you would like to enable your new Nextcloud integration.
 
-To activate the module please go to the Project settings in the desired project and access the Modules menu entry in the side menu, there you can activate the module "File storages". Once this is done a new lateral menu entry called "File storages" at the bottom of the menu where you can select which file storage integration you want to activate in this project.
+To activate the module, go to the *Project settings* in the desired project and access the Modules menu entry in the side menu. There, activate the "File storages" module. Once this is done, a new menu entry called "File storages" appears at the bottom, where you can select your new Nextcloud integration.
```
Hi @psatyal,

I have the feeling that removing the part of the menu makes the sentence unclear (the bottom of the content? bottom of the complete screen?) I would like to keep the menu or the side menu there to make it clear.

psatyal replied:
Hi @marcalcobe, just fixed this in a separate commit.

marcalcobe reviewed (commented):
Hi @psatyal,
Thank you so much for the changes on the phrasing. I included some small details that I'm not sure we should change from a text perspective.

psatyal committed:
Minor grammatical fixes

psatyal committed:
Minor update

psatyal committed:
Forgot to add this

psatyal commented:
Hi @marcalcobe. Made changes based on your comments and approved them on my end.

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the market-leading collaboration platform, and OpenProject, the leading open source project management software.
```
Hmm, "market leading collaboration platform". Isn't OpenProject a collaboration platform, too? Shouldn't we focus on the file sharing/serving/hosting bit of Nextcloud?

marcalcobe replied:
Hi @wielinde. This. is related with how we describe the integration everywhere. Let's discuss it with @birthe so we are all in the same page and I will update it afterwards.

psatyal replied:
Agreed, I'd recommend the same phrasing as the other doc: "It combines the strengths of [Nextcloud](https://nextcloud.com/), the world’s most-deployed on-premises collaborative file storage platform and [OpenProject](https://www.openproject.org/), the leading free and open source project management and collaboration software."

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the market-leading collaboration platform, and OpenProject, the leading open source project management software.
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly in their Nextcloud instance.
+The integration enables users to keep an eye on ongoing project activities directly in their Nextcloud instance and link their Nextcloud files to OpenProject work packages. To learn more about how to use the integration, please refer to the [Nextcloud user guide](../../../user-guide/nextcloud-integration/)
```
If  with "keep an eye on the ongoing project activities" you mean the dashboard, then I would deprioritize that aspect and focus more on "Where are the files that I need to be able to complete a task?" and "Where do I put the final files after completing a task?".

marcalcobe replied:
Hi @wielinde. Sounds good to me. Also might change after we agree on a common description.

psatyal replied:
Agreed, will update.

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+## Step-by-step setup instructions
 
-The first step of the combined effort is the integration of OpenProject in the Nextcloud dashboard. Users can add an OpenProject widget to display latest changes to project's work packages. With this, it offers users a view of ongoing projects and activities.
+The integration is available starting with Nextcloud 20 and OpenProject 12.2. To be able to configure this integration, you need to be an administrator in both Nextcloud and OpenProject instances.
```
For integrating with OpenProject 12.2 the minimal version of Nextcloud is 22. The minimal version version of the Nextcloud app is 2.0.0

marcalcobe replied:
I will update this information also in this documentation page.

psatyal replied:
I'll add this information

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+The integration is available starting with Nextcloud 20 and OpenProject 12.2. To be able to configure this integration, you need to be an administrator in both Nextcloud and OpenProject instances.
 
-## Step by step instructions
+**1. Add OpenProject integration app and connect the instance** - in Nextcloud
```
```suggestion
**1. Add the "OpenProject integration" app and connect the instance** - in Nextcloud
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+**1. Add OpenProject integration app and connect the instance** - in Nextcloud
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly out of their Nextcloud instance.
+To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store under your user name in _Your apps_. You can use the search field in the top right corner to look for OpenProject integration app. Click the button **Download and enable**.
```
What do you mean with "under your user name"? Do you mean the menu that opens when you click your user avatar on the top right corner in Nextcloud?

marcalcobe replied:
Hi @wielinde. I didn't touch this step as it was coming from the previous documentation but i updated it now.

psatyal replied:
Adding the word "menu" to make avoid potential confusion.

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+![Nextcloud_app_store](Nextcloud_app_store.png)
 
-To activate your integration to OpenProject in Nextcloud, navigate to the built in app store under your user name in Your apps. You can use the search field in the top right corner to look for the OpenProject integration. Click the button Download and enable.
+Once the OpenProject integration app is downloaded and enabled, you can access it through the side menu and enter the host URL of your desired OpenProject instance.
```
Might be important to mention that you find it within the settings.

marcalcobe replied:
I will add this to the documentation as its much clearer.

psatyal replied:
Added this precision.

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
The Nextcloud version was still 20 instead of 22

psatyal committed:
Updated based on wieland's feedback

psatyal commented:
Updated the text based on @wielinde's comments.

marcalcobe reviewed (commented):
All the changes has been done on the text. The only pending text would be the unified description of the integration in case is needed on the top of the documentation page.
Also a review from any @opf/doc-writers would be appreciated as double check.

marcalcobe committed:
Review the language (2nd person) and styles

I double checked everything is in second person "you" and reviewed the styles on how we emphasise buttons, links, notes and sections in other places of the documentation guides and unified this page.

psatyal committed:
Updated language for clarity and grammar

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the world’s most deployed on-premises collaborative file storage platform and OpenProject, the leading free and open source project management and collaboration software
```
@marcalcobe @psatyal Tiny detail: What is the right genitive tick in English? The one used here is not the one I have on my US English keyboard. Seems suspicious to me. 

```suggestion
The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the world's most deployed on-premises collaborative file storage platform and OpenProject, the leading free and open source project management and collaboration software
```

psatyal replied:
@wielinde Indeed, the one you've replaced it with is more standard; don't know where the original came from. Commiting. (For reference: I have an ANSI keyboard to with an "ABC - Extended" setup). (Edited)

psatyal committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Actually your apostrophe is better.

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+## Step-by-step setup instructions
 
-The first step of the combined effort is the integration of OpenProject in the Nextcloud dashboard. Users can add an OpenProject widget to display latest changes to project's work packages. With this, it offers users a view of ongoing projects and activities.
+For integrating with OpenProject 12.2, the minimum required version of Nextcloud is 22. For the desktop Nextcloud application, the minimum supported version is 2.0.0. To be able to configure this integration, you need to have administrator privileges in both your Nextcloud and OpenProject instances.
```
AFAIK Nextcloud calls their plugins "Apps" and not "applications". 

```suggestion
For integrating with OpenProject 12.2, the minimum required version of Nextcloud is 22. For the Nextcloud app  "OpenProject integration" that extends your Nextcloud installation, the minimum required version is 2.0.0. To be able to configure this integration, you need to have administrator privileges in both your Nextcloud and OpenProject instances.
```

psatyal replied:
Ah, good point @wielinde. I would remove the comma after 

> integration",

Looks like I can modify your suggested change, let me try it.

psatyal committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+**1. Add the "OpenProject integration" app and connect the instance** - in Nextcloud
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly out of their Nextcloud instance.
+To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store in the menu under your user name by clicking on _Your apps_. You can use the search field in the top right corner to look for the OpenProject integration app. Click the button **Download and enable**.
```
There is no user name in the Nextcloud header. There is only an avatar.

```suggestion
To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store. You find the app store in the menu that opens when you click your round user avatar in the top right corner. In the app store click on _Integrations_. There you will find the "OpenProject integration" app. You can also use the search field in the top right corner to look for the OpenProject integration app. Once you have found the app, click the button **Download and enable**.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+![Nextcloud_app_store](Nextcloud_app_store.png)
 
-To activate your integration to OpenProject in Nextcloud, navigate to the built in app store under your user name in Your apps. You can use the search field in the top right corner to look for the OpenProject integration. Click the button Download and enable.
+Once the OpenProject integration app is downloaded and enabled, you can access it from within the Settings page, via the the side menu. Start by entering the *host URL* of your desired OpenProject instance.
```
```suggestion
Once the OpenProject integration app is downloaded and enabled, you can access it from within the _Settings_ page, via the side menu. Start by entering the _Host URL_ of your desired OpenProject instance.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
-![Nextcloud_app_store](Nextcloud_app_store.png)
+![Nextcloud_host](3_2_01 -NC_Step_1.png)
+
+Click on the **Save** button and open your OpenProject instance in a new tab.
```
```suggestion
Click on the **Save** button and open your OpenProject instance in a new browser tab.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+
+![Storage_index](3_0_00-OP_OAuth_Empty_Index.png)
+
+By default, the storage *provider type* is set to Nextcloud; this does not need to be modified. To proceed, enter a *name* to your Nextcloud storage. This will be visible to all the users using it. We highly recommended choosing a clear and distinct name that allows users to differentiate it from other potential file storages integrations in the future. You will also need to set the *Host URL* of ...
```
```suggestion
By default, the storage *Provider type* is set to _Nextcloud_; this does not need to be modified. To proceed, enter a *Name* to your Nextcloud storage. This name will be visible to all the users using it. We highly recommended choosing a clear and distinct name that allows users to differentiate it from other potential file storages integrations in the future. You will also need to set the *Host URL* of your Nextcloud instance. The *Host URL* is the web address of your Nextcloud instance. Make sure you include the _https://_ bit of the address.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+
+![Storage_basic_information](3_0_01-OP_General_Info.png)
+
+Once you fill in the mandatory fields, click on the **Save and continue setup** button.
```
```suggestion
Once you filled in all mandatory fields, click on the **Save and continue setup** button.
```

psatyal replied:
Just a note here: "fill in" is gramattically correct, you do not use the past tense with when the sentence begins with "once (something something). (I don't know why, actually 🤔)... So we should revert if commited :) @marcalcobe, @wielinde.

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+
+**3. Generate and copy the OpenProject OAuth values** - in OpenProject and Nextcloud
+
+In this step, the *OpenProject OAuth values* are generated automatically. These values are needed to permit OpenProject to connect to Nextcloud. You will need to copy them from the Nextcloud integration window in a new tab (without exiting or closing your OpenProject tab) and pasting them in the respective fields under _OpenProject OAuth settings_.
```
```suggestion
In this step, the *OpenProject OAuth values* are generated automatically. These values are needed to permit OpenProject to connect to Nextcloud. If you have not done so yet, we recommend you to best have two browser tabs open, the current one with OpenProject and the former one with Nextcloud. Now copy the two *OpenProject OAuth values* from the OpenProject tab to the respective fields in the Nextcloud tab. Simply paste them into the respective _OpenProject OAuth settings_ fields.
```

psatyal replied:
"We recommend you have" (vs. "to best have").

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+In this step, the *OpenProject OAuth values* are generated automatically. These values are needed to permit OpenProject to connect to Nextcloud. You will need to copy them from the Nextcloud integration window in a new tab (without exiting or closing your OpenProject tab) and pasting them in the respective fields under _OpenProject OAuth settings_.
 
-**Activate the OpenProject integration app**
+> **Important**: The *OpenProjects OAuth values (client ID and client secret)* are not accessible again after you close the window. Please make sure you copy the generated values you see in the _Nextcloud OpenProject Integration settings_.
```
```suggestion
> **Important**: The *OpenProjects OAuth values (client ID and client secret)* are not accessible again after you close the window. Please make sure you copy the generated values you see in the _Nextcloud OpenProject Integration settings_. Treat these values with care. Do not save them in any public place. Simply treat them as carefully as you would treat an important password.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+**4. Generate and copy the Nextcloud OAuth values** - in Nextcloud and OpenProject
 
-![OpenProject_API_key_copy](OpenProject_API_key_copy.png)
+As in the previous step, the *OAuth values* are generated automatically, but this time on the Nextcloud end. These values are needed to allow the connection from Nextcloud to OpenProject, so you will need to once again copy them here and paste them in OpenProject (without closing the Nextcloud tab). If you had not clicked the **Done. Continue setup** button in the previous step, you can do so now ...
```
```suggestion
As in the previous step, the *OAuth values* are generated automatically, but this time on the Nextcloud end. These values are needed to allow the connection from Nextcloud to OpenProject, so you will need to once again copy them here and paste them in OpenProject (without closing the Nextcloud tab). If you had not clicked the **Done. Continue setup** button in the OpenProject tab in the previous step, you can do so now to proceed to the screen where you will be able to paste the Nextcloud OAuth values in OpenProject.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+**5. Enable the file storages module and activate it in the desired project** - in OpenProject
 
-The link will show here:
+Now that the integration setup is complete and ready to use, there is just one more step for you as administrator: you need to activate the *File storages* module and specify for which projects you would like to enable your new Nextcloud integration.
```
```suggestion
Now that the integration setup is complete and ready to use, there is just one more step for you as administrator: in every project in which you want to use the integration you need to activate the *File storages* module and add your recently configured file storage to the project.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+## How to delete a file storage integration
 
-If the notifications are not displayed in your Nextcloud dashboard, please check the following in your Nextcloud basic settings: in the background jobs, Cron must be activated.
+As an administrator, you can always delete the integration to Nextcloud using the **Delete** button in OpenProject settings or the **Reset** button in Nextcloud settings.
```
```suggestion
As an administrator, you can always delete the integration to Nextcloud using the **Delete** button in  the OpenProject settings or the **Reset** button in the Nextcloud settings.
```

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
 
-![Cron_job_settings](Cron_job_settings.png)
\ No newline at end of file
+> **Important:** If you perform this action, the integration will be reset and deleted and all settings and user connections that were created will be deleted. This means that should you want to reconfigure the integration, you will need complete the entire setup process once again.
```
@marcalcobe Hmm, the delete in OpenProject deletes all file links. The reset button in Nextcloud just deletes the connections but not the links. We may want to be very clear what does what. So we better invest these lines here.

marcalcobe replied:
@wielinde totally agree with this quote add. Thanks.

wielinde commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,96 @@
+![module_activation](Settings_modules.png)
 
-![Unified_search](Unified_search.png)
+![settings_files_storages](Settings_files_storages.png)
```
@marcalcobe Here we could link to the user guide, in the sense of "All further steps are explained in the User guide"

marcalcobe replied:
Hi @wielinde, @psatyal added this information in his last commit.

wielinde reviewed (commented):
Hey @marcalcobe I added comments and suggestions. Very often I added more details and text to reduce ambiguity and confusion. Please use whatever you feel is an improvement.

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Apply minor text changes to the proposed text

The last sentence had a "the" that was not needed an the verb fresh sounds a little bit to casual in my opinion.

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

marcalcobe committed:
Update docs/system-admin-guide/integrations/nextcloud/README.md

Co-authored-by: Wieland Lindenthal <w.lindenthal@forkmerge.com>

psatyal committed:
Minor + link to user guide

Made two minor changes (grammmatical) and added a link to the user guide at the end

marcalcobe commented:
@opf/doc-writers, @birthe or @as-op all the relevant changes and reviews has been done from our side. Can you please do the last double check before we merge it to 12.2?

as-op reviewed (changes requested):
* There are spaces in the image filenames, which we unfortunately need to avoid:
"3_2_01 -NC_Step_1.png"
"3_2_06 -NC_OP_OAuth_Replace.png"
"3_4_00 - OAuth - OP Success.png"

* Please delete the "_old" image folder. (Old images are always available through git)

as-op commented on docs/system-admin-guide/integrations/nextcloud/README.md:
```diff
@@ -1,79 +1,102 @@
 
-If the notifications are not displayed in your Nextcloud dashboard, please check the following in your Nextcloud basic settings: in the background jobs, Cron must be activated.
+Once the file storage is added and enabled for projects, your users are able to take full advantage of the Nextcloud-OpenProject integration. For more information on how link Nextcloud files to work packages in OpenProject and access linked work packages in Nextcloud, please refer to the [Nextcloud integration user guide](../../../user-guide/nextcloud-integration/).
+.
```
There's an unhappily single dot. :)

marcalcobe replied:
The two things are already solved. Thanks @as-op

marcalcobe committed:
Image details solved and old folder deleted

Resolved the problems with the spaces in images names and the OLD folder

marcalcobe committed:
Floating dot deleted

as-op reviewed (changes requested):
Sorry, one more issue. Just noticed it. In this PR there is also a stub for the [user-guide/nextcloud-integration/README.md](https://github.com/opf/openproject/blob/c64c52b16869210d685e45fa2094fd939b8fb52c/docs/user-guide/nextcloud-integration/README.md).
But it's also topic in PR https://github.com/opf/openproject/pull/11016 - [README.md](https://github.com/opf/openproject/blob/7279d1f7270e464c1c247c61244cddf8900374c9/docs/user-guide/nextcloud-integration/README.md).
Since it's a new file in both PR, it's possible and best to delete it in this PR to avoid a merge conflict.
Thanks!

marcalcobe committed:
Delete the related file with #11016

Delete the related file with #11016 to avoid conflict when merge

marcalcobe commented:
Hi @as-op,
I've deleted the file from here.

as-op commented:
Recreated & merged to release/12.2 at https://github.com/opf/openproject/pull/11062
